MapleServer2/PacketHandlers/Game/FieldEnterHandler.cs

```diff
@@ -56,5 +59,9 @@ public override void Handle(GameSession session, PacketReader packet)
 
         List<GameEvent> gameEvents = DatabaseManager.Events.FindAll();
         session.Send(GameEventPacket.Load(gameEvents));
+
+        // get map id as string and size of 8 digits
+        string mapId = player.MapId.ToString().PadLeft(8, '0');
+        player.TrophyUpdate(type: "map", addAmount: 1, code: mapId);
```

Comment:
You might have already seen this - but the "map" category is actually pretty small, I get the impression that it was used early on and then abandoned in favour of the "explore" category. Not something to worry about for this PR though :)